addon/ng2/tasks/npm-install.js

```diff
@@ -0,0 +1,17 @@
+/* jshint node: true, esversion: 6 */
```

Comment:
I think in this case it's okay, since it's not our code. Unless you see a problem caused by their code.